apps/condo/domains/user/integration/sberid/SberIdIdentityIntegration.js

```diff
@@ -0,0 +1,174 @@
+const axios = require('axios').default
+
+// eslint-disable-next-line import/order
+const https = require('https')
+const jwtDecode = require('jwt-decode')
+const { isNil } = require('lodash')
+const { customAlphabet } = require('nanoid')
+
+
+const conf = require('@open-condo/config')
+
+// get sber id configuration params
+const SBER_ID_CONFIG = process.env.SBER_ID_CONFIG ? JSON.parse(process.env.SBER_ID_CONFIG) : {}
+const {
+    authorizeUrl,
+    tokenUrl,
+    userInfoUrl,
+    scope,
+    clientId,
+    clientSecret,
+    redirectUri,
+    cert,
+    key,
+    verifyServerSsl,
+} = SBER_ID_CONFIG
+const callbackPath = '/api/sber_id/auth/callback'
+const callbackUri = redirectUri || `${conf.SERVER_URL}${callbackPath}`
+const axiosTimeout = 10000
+
+// instantiate httpsAgent in order to support mTLS communication with sber id servers
+const httpsAgent = new https.Agent({
+    rejectUnauthorized: verifyServerSsl,
+    cert,
+    key,
+})
+
+// instantiate request id generator
+const nanoid = customAlphabet('1234567890abcdef', 32)
```

Comment:
can we use uuid() or generators from 'openid-client' ?

Reply:
Unfortunately nope, since uuid and generators from openid-client doesn't support to generate unique identifiers based on specific alphabet (since this functionality is out of openid scope)

Reply:
can you use getRandomString?